Constitution.md

```diff
@@ -207,6 +225,42 @@ If there are no written minutes, written notice, and/or the information presente
 
 If an executive officer is dismissed, the club must follow [**Section 3: Club Changeover**](#section-3-club-changeover) in order to follow the proper procedure(s) outlined to fill the vacant position(s).
 
+## Article 2d: Committees, Councillors, and Community Advocates
+
+### Committees
+A Committee is a group of council members that forms to complete a well-defined goal. Any Executive Officer may propose a committee by outlining its purpose and the criteria of its members (if any). The Council will vote on the formation of this Committee, and upon its formation, the proposing Executive Officer will lead it.
+
+A Committee leader will have the following duties:
+1. Appoint Committee members from Council.
+2. If applicable, announce the formation of the Committee to Members of the Association so that they may apply for it.
```

Comment:
nit: should "Members of the Association" be capitalized? maybe just "members of the CSSA"?